Socket.php

```diff
@@ -49,21 +49,201 @@
 class Socket extends HoaSocket
 {
     /**
+     * Entity type: isuser.
+     *
+     * @const string
+     */
+    const ENTTYPE_ISUSER = 'isuser';
```

Comment:
Please, do not use shortcuts or acronyms, e.g. `USER_ENTITY`, `CHANNEL_ENTITY` would be fine I guess.
Also, I am not sure they are “entities”, but more things like “parts”. Is entity the wording of the RFC? If yes, keep it.